zwave_js_server/model/controller/__init__.py

```diff
@@ -420,13 +420,16 @@ async def async_is_failed_node(self, node_id: int) -> bool:
         return cast(bool, data["failed"])
 
     async def async_get_association_groups(
-        self, node_id: int
+        self, source: Association
```

Comment:
I realize you didn't create this class, but since you are cleaning up, `Association` should probably renamed to `AssociationAddress` to match the zwave-js project